shell/platform/android/flutter_main.cc

```diff
@@ -2,6 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+#include "common/settings.h"
+#include "impeller/base/validation.h"
 #define FML_USED_ON_EMBEDDER
```

Comment:
@jonahwilliams You are running into the message loop error above because this define is not at the top.

Reply:
🤦 Thanks!

Reply:
While you are at it, https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes for the include order. These includes would go below the related header.

Reply:
Done